c2cgeoportal/scaffolds/create/print/print-apps/+package+/config.yaml.mako

```diff
@@ -87,31 +87,88 @@ templates:
                         urlExtractor: (.*)
                         urlGroup: 1
 
-#    A4 landscape: !template
-#        reportTemplate: A4_Landscape.jrxml
-#        attributes: &attributes
-#            title: !string
-#                default: ""
-#            comments: !string
-#                default: ""
-#            debug: !boolean
-#                default: false
-#            legend: !legend {}
-#            northArrow: !northArrow
-#                size: 40
-#                default:
-#                    graphic: "file:///north.svg"
-#            scalebar: !scalebar
-#                width: 150
-#                height: 20
-#                default:
-#                     fontSize: 8
-#            map: !map
-#                maxDpi: 254
-#                dpiSuggestions: [254]
-#                zoomLevels: !zoomLevels
-#                    scales: [100, 250, 500, 2500, 5000, 10000, 25000, 50000, 100000, 500000]
-#                width: 555
-#                height: 675
-#            datasource: !datasource *datasource
-#        processors: *processors
+    2 A4 landscape: !template
```

Comment:
I remember each time I open it :p but I have no clue about how it can help me :) but I understand that there is too much parameters that does'nt change for each new template.